api/test/functions/ucan-invocation-router.test.js

```diff
@@ -0,0 +1,42 @@
+import test from 'ava'
+import { GenericContainer as Container } from 'testcontainers'
+
+import { parse } from '@ipld/dag-ucan/did'
+import { CAR } from '@ucanto/transport'
+
+import getServiceDid from '../../authority.js'
+import { handler } from '../../functions/ucan-invocation-router.js'
+
+import { alice } from '../fixtures.js'
+
+test.before(async t => {
+  await new Container('amazon/dynamodb-local:latest')
+    .withExposedPorts(8000)
+    .start()
+})
+
+// TODO: Need to set ENV for dbEndpoint...
+test.skip('ucan-invocation-router', async (t) => {
```

Comment:
@olizilla should we just drop this test to the lambda directly?

It is quite hard to inject the needed resources here... I would suggest follow up PR with more integration kind of tests, where we run the API Stack with all routes and perform tests using HTTP Calls.